plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcFileWriterFactory.java

```diff
@@ -223,21 +216,4 @@ public static OrcDataSink createOrcDataSink(TrinoFileSystem fileSystem, Location
     {
         return OutputStreamOrcDataSink.create(fileSystem.newOutputFile(location));
     }
-
-    private static CompressionKind getCompression(Properties schema, JobConf configuration)
-    {
-        String compressionName = OrcConf.COMPRESS.getString(schema, configuration);
-        if (compressionName == null) {
```

Comment:
what was the case when we got `null` here? Was that really possible - how is that case handled now?

Reply:
I don't think that was possible before, but may have been when this code was originally written. Or possibly it was just being paranoid. I checked the history and this has been there since the new ORC writer first integrated.